templates/header.jsx

```diff
@@ -49,7 +50,7 @@ export default function Header(props) {
           <div className="js-heading" ref={jsxHeading}></div>
           }
 
-          <div className={prefixClasses(classNamePrefixes, ['__title-inner'])} aria-hidden={!_disableAccessibilityState}>
+          <div className={prefixClasses(classNamePrefixes, ['__title-inner'])} aria-hidden={isIOSDevice ? false : !_disableAccessibilityState}>
```

Comment:
Hi Eleanor,
Adapt.device already has a function to perform this check. It may be best to use this in case it ever gets improved in the future

Reply:
`<div className={prefixClasses(classNamePrefixes, ['__title-inner'])} aria-hidden={Adapt.device.isAppleDevice() ? false : !_disableAccessibilityState}>`

Reply:
The only thing about using `Adapt.device.isAppleDevice()` is that Ive noticed before that it does not apply to iPad devices and will need to add an additional check to isAppleDevice, such as `navigator.userAgent.match(/Mac/) && navigator?.maxTouchPoints > 2;` to account for iPadOS. I will create another PR for this in the core repo now.

Reply:
I have opened another issue to update `isAppleDevice` here https://github.com/adaptlearning/adapt-contrib-core/issues/75 and have included that fix in this pull request.